assets/sass/06-components/pagination.scss

```diff
@@ -46,6 +46,17 @@
 		display: inline-block;
 		fill: currentColor;
 		vertical-align: middle;
+		position: relative;
+	}
+
+	.nav-previous .svg-icon,
+	.prev .svg-icon {
+		top: -3px;
```

Comment:
This is incredibly nitpicky, but can you update this to `-2px`?

Reply:
```suggestion
		top: -2px;
```

Reply:
@melchoyce you can actually suggest changes like that to make it super easy for authors to commit them by clicking this icon in the comment toolbar:

![Screenshot from 2020-10-12 21-54-16](https://user-images.githubusercontent.com/3902039/95789479-842c5f80-0cd5-11eb-841a-456a679e1b77.png)

Reply:
Oh rad, thanks for the tip 👍